src/Microsoft.DotNet.Cli.Utils/DepsJsonCommandFactory.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using NuGet.Frameworks;
+using System.IO;
+using System;
+using Microsoft.DotNet.ProjectModel;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public class DepsJsonCommandFactory : ICommandFactory
+    {
+        private DepsJsonCommandResolver _depsJsonCommandResolver;
+        private string _temporaryDirectory;
+        private string _depsJsonFile;
+        private string _runtimeConfigFile;
+        private string _nugetPackagesRoot;
```

Comment:
:smile: Yup, don't know why I thought I was making the resolver in the Create method